k8s/scripts/provision.sh

```diff
@@ -152,12 +152,12 @@ kind: InitConfiguration
 ---
 apiServerExtraArgs:
   enable-admission-plugins: Initializers,NamespaceLifecycle,LimitRanger,ServiceAccount,DefaultStorageClass,DefaultTolerationSeconds,NodeRestriction,MutatingAdmissionWebhook,ValidatingAdmissionWebhook,ResourceQuota
-  feature-gates: "BlockVolume=true,CSIBlockVolume=true,VolumeSnapshotDataSource=true"
+  feature-gates: "CSIBlockVolume=true,VolumeSnapshotDataSource=true"
   allow-privileged: "true"
   runtime-config: admissionregistration.k8s.io/v1alpha1
 apiVersion: kubeadm.k8s.io/v1alpha3
 controllerManagerExtraArgs:
-  feature-gates: "BlockVolume=true,,CSIBlockVolume=true,VolumeSnapshotDataSource=true"
+  feature-gates: "CSIBlockVolume=true,VolumeSnapshotDataSource=true"
```

Comment:
is there a reason why BlockVolume was removed?

Reply:
Reverted